src/NpmUtilities.js

```diff
@@ -47,8 +47,15 @@ export default class NpmUtilities {
   }
 
   @logger.logifyAsync()
-  static publishTaggedInDir(tag, directory, callback) {
-    ChildProcessUtilities.exec("cd " + escapeArgs(directory) + " && npm publish --tag " + tag, null, callback);
+  static publishTaggedInDir(opts, directory, callback) {
+    const validOpts = ["tag", "registry"];
+    const args = Object.keys(opts).map((argKey) => {
+      if (validOpts.indexOf(argKey) > -1) {
+        return `--${escapeArgs(argKey)} ${escapeArgs(opts[argKey])}`;
+      }
+    });
+    let command = ("npm publish " + args.join(" ")).trim();
+    ChildProcessUtilities.exec("cd " + escapeArgs(directory) + " && " + command, null, callback);
```

Comment:
I'm not a huge fan of this argument whitelisting, though I understand the purpose.

Instead of direct flags, I would rather provide config like this via [env vars](https://docs.npmjs.com/misc/config#environment-variables) passed in the options argument.

```
const command = ("npm publish --tag " + tag).trim()
const env = {
  npm_config_registry: opts.registry
};
ChildProcessUtilities.exec("cd " + escapeArgs(directory) + " && " + command, { env }, callback);
```